ACT Landing Page.md

```diff
@@ -0,0 +1,173 @@
+# MITRE Adaptive Capabilities Testing (ACT)™
+
+## Transform Your Cybersecurity Assessment Approach
+
+MITRE ACT™ is a flexible, efficient, and customizable security and risk assessment framework that empowers organizations to make informed, risk-based decisions about their cybersecurity posture. Unlike traditional compliance-driven assessments, ACT focuses on understanding the actual risks to your mission-critical capabilities, helping you allocate resources where they matter most.
+
+---
+
+## What is MITRE ACT?
+
+MITRE Adaptive Capabilities Testing (ACT) is a capabilities-focused assessment framework that encourages and drives risk-based decision-making by considering all available Risk Information Sources (RIS). The findings generated by compliance-focused assessments are valuable, but when considered in isolation they lack the context necessary to determine how they actually impact the organization. MITRE ACT helps organizations move beyond checkbox compliance to understand what their security findings actually mean in the context of their mission objectives... and whether or not compliance is the "right" choice.
+
+### The ACT Difference
+
+While traditional assessments, such as Security Control Assessments (SCAs), generate valuable lists of low-level technical findings, ACT **transforms** this data into **actionable intelligence**:
+
+- **Risk-driven rather than compliance-driven**: ACT focuses on risk identification and analysis at the capability level, de-emphasizing technical findings and compliance checkboxes
+- **Capability-oriented rather than control-oriented**: Capabilities state objectives, while controls state specific implementation requirements that might help meet those objectives
+- **Based on multiple Risk Information Sources**: ACT considers **all available risk data** at the time of assessment, not just current compliance status
+- **More understandable and actionable**: ACT adds context, brings conversation to a higher level, and helps readers determine **what should be done**
+
+---
+
+## Why ACT Matters
+
+### The Problem with Traditional Assessments
+
+Many federal and commercial cybersecurity programs suffer from similar weaknesses:
+
+1. Low-level technical findings are handed directly to less-technical decision makers
+2. Decision-makers lack an appropriate framework for understanding the actual risks
+3. Without risk context, "close all findings" becomes the default reaction - **regardless of cost or impact to the organization**
```

Comment:
Maybe we can add another bullet for this thought:
without risk context, just closing all the findings gets the system to a temporary compliance state without understanding the root cause and addressing it to prevent similar findings from reoccurring